packages/nouns-sdk/src/contract/addresses.json

```diff
@@ -12,27 +12,27 @@
     "nounsDAOLogicV1": "0xa43aFE317985726E4e194eb061Af77fbCb43F944"
   },
   "4": {
-    "nounsToken": "0x632f34c3aee991b10D4b421Bc05413a03d7a37eB",
```

Comment:
@solimander is it important to keep the original values here before merging to master?